ranger-http-client/src/test/java/io/appform/ranger/client/http/BaseRangerHttpClientTest.java

```diff
@@ -70,21 +73,21 @@ public void startTestCluster() throws Exception {
                 ))
                 .build();
         val response = objectMapper.writeValueAsBytes(responseObj);
-        server.stubFor(get(urlEqualTo("/ranger/services/v1"))
+        wireMockExtension.stubFor(get(urlEqualTo("/ranger/services/v1"))
                 .willReturn(aResponse()
                         .withBody(response)
                         .withStatus(200)));
 
         httpClientConfig = HttpClientConfig.builder()
                 .host("127.0.0.1")
-                .port(server.port())
+                .port(wireMockExtension.getPort())
                 .connectionTimeoutMs(30_000)
                 .operationTimeoutMs(30_000)
                 .build();
         log.debug("Started http subsystem");
     }
 
-    @After
+    @AfterEach
     public void stopTestCluster() {
```

Comment:
lol .. rename

Reply:
This is actually not needed. There is nothing happening afterEach, renamed the beforeEach to prepareHttpMocks.